spec/models/report_spec.rb

```diff
@@ -1153,6 +1153,12 @@ class ReportInitError < StandardError
         freeze_time DateTime.parse("2017-03-01 12:00")
 
         ip = [81, 2, 69, 142]
+        GlobalSetting.class_eval do
```

Comment:
no need to do this, this should do the trick... 

`global_setting "maxmind_license_key", ""`

Reply:
Thanks! That looks much cleaner indeed.